index.js

```diff
@@ -1,7 +1,15 @@
 'use strict';
 const Build = require('./lib/build');
+const factory = require('./lib/factory');
 const Job = require('./lib/job');
 const Pipeline = require('./lib/pipeline');
 const User = require('./lib/user');
 
-module.exports = { Build, Job, Pipeline, User };
+module.exports = {
+    Build,
+    configureBuildModel: factory.configureBuildModel,
```

Comment:
Ideally, it would be great to configure all of it in one place like:
```js
const models = require('screwdriver-models');
const Build = models.Build;
models.configure({ datastore, executor, key });

const build = new Build();
```